### DISCUSSION TITLE
Set Visibility to Filter Columns

### DISCUSSION THREAD

**Discussion by mdomas1** [dotnet/DataGridExtensions#56] [Q&A] [answered]
Hello All, 

I want to have the visibility of the Filter off on the first two rows of my datagrid, what would be the proper code to accomplish this?

I've tried: 
`dataGrid1.ItemsSource = new ListCollectionView(datat.DefaultView);`
            `dataGrid1.Columns[0].SetIsFilterVisible = False;`
            `dataGrid1.Columns[1].SetIsFilterVisible = False;`
And
`dataGrid1.Columns[0].IsFilterVisible = False;`
And
`dataGrid1.Columns[0].SetIsFilterVisible = IsFilterVisible.False;`

Plz help

**Comment by tom-englert** [comment, marked as answer]
https://github.com/dotnet/DataGridExtensions/blob/497f600f69229c57797323826817c5e46e658322/src/DataGridExtensionsSample/Views/Customized2View.xaml#L129-L134

**Reply by mdomas1** [reply]
Thank you for the reply! Is there any way to do it programmatically with C#?

**Reply by tom-englert** [reply]
`dataGrid1.Columns[0].SetIsFilterVisible(false);`
